### PR TITLE
Moved common modifiers to own contract. Extensions now inherit this …

### PR DESCRIPTION
**Pull request by bweick**
…contract.

### CODE REVIEW

File: contracts/core/extensions/CoreInternal.sol [hunk 1]
```diff
@@ -16,8 +16,9 @@
 
 pragma solidity 0.4.24;
 
-import { Ownable } from "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import { CoreSharedModifiers } from "../lib/CoreSharedModifiers.sol";
```
Comment (felix2feng) [inline contracts/core/extensions/CoreInternal.sol]: What's the difference between `lib` and `extension`?

File: contracts/core/extensions/CoreInternal.sol [hunk 2]
```diff
@@ -16,8 +16,9 @@
...
 import { CoreState } from "../lib/CoreState.sol";
+import { Ownable } from "zeppelin-solidity/contracts/ownership/Ownable.sol";
```
Comment (asoong) [inline contracts/core/extensions/CoreInternal.sol]: Absolute paths tend to be kept at the top

File: contracts/core/extensions/CoreIssuance.sol [hunk 3]
```diff
@@ -16,11 +16,12 @@
 
 pragma solidity 0.4.24;
 
-import { SafeMath } from "zeppelin-solidity/contracts/math/SafeMath.sol";
+import { CoreSharedModifiers } from "../lib/CoreSharedModifiers.sol";
 import { CoreState } from "../lib/CoreState.sol";
 import { ISetToken } from "../interfaces/ISetToken.sol";
 import { ITransferProxy } from "../interfaces/ITransferProxy.sol";
 import { IVault } from "../interfaces/IVault.sol";
+import { SafeMath } from "zeppelin-solidity/contracts/math/SafeMath.sol";
```
Comment (asoong) [inline contracts/core/extensions/CoreIssuance.sol]: Same

File: contracts/core/lib/CoreSharedModifiers.sol [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+/**
+ * @title Core Shared Modifiers
+ * @author Set Protocol
+ *
+ * The Core Shared Modifiers library contains the modifiers that are shared across the different
+ * Core extensions.
+ */
+
+contract CoreSharedModifiers is
```
Comment (asoong) [inline contracts/core/lib/CoreSharedModifiers.sol]: WYDT about just `CoreModifiers`

**Pull request merged**